.github/workflows/diff.yaml

```diff
@@ -275,11 +275,13 @@ jobs:
       - name: Run stress test (plain)
         run: |
           cd tests/stress
+          source ve3/bin/activate
           ./continuous_integration
 
       - name: Run stress test (SSL)
         run: |
           cd tests/stress
+          source ve3/bin/activate
```

Comment:
Not sure why this is required, but it's probably required in the daily workflows `Release Debian` because the stress tests are run there as well